egs/csj/s5/local/csj_make_trans/csj_automake.sh

```diff
@@ -5,113 +5,161 @@
 # Apache 2.0
 # Acknowledgement  This work was supported by JSPS KAKENHI Grant Number 26280055.
 
-if [ $# -ne 2 ]; then
-  echo "Usage: "`basename $0`" <speech-dir> <transcription-dir> "
+if [ $# -ne 3 ]; then
+  echo "Usage: "`basename $0`" <speech-dir> <transcription-dir> <csj-version>"
   echo "See comments in the script for more details"
```

Comment:
At least a bit of help (on the mandatory arguments at least) would be prefered. Especially the csj-version is not clear.